js/grumbles/form.directive.js

```diff
@@ -21,9 +21,13 @@
       },
       link: function(scope, element, attributes){
         if(!scope.grumble) scope.grumble = new GrumbleFactory();
-        scope.ifSaveSuccessful = function(response){
-          console.log(response);
-          $state.go($state.current, {}, {reload: true});
+        scope.ifCreateSuccessful = function(response){
+          scope.$parent.GrumbleIndexViewModel.grumbles.push(response);
```

Comment:
Not even sure we need the `$parent` since we are not nesting VMs: [source](http://stackoverflow.com/questions/21453697/angularjs-access-parent-scope-from-child-controller)